[2024-01-17 00:31:52,769] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_redshift.verificar_accesos manual__2024-01-17T00:31:47.666927+00:00 [queued]>
[2024-01-17 00:31:52,806] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_redshift.verificar_accesos manual__2024-01-17T00:31:47.666927+00:00 [queued]>
[2024-01-17 00:31:52,808] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-01-17 00:31:52,819] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2024-01-17 00:31:52,820] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-01-17 00:31:53,014] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): verificar_accesos> on 2024-01-17 00:31:47.666927+00:00
[2024-01-17 00:31:53,018] {standard_task_runner.py:52} INFO - Started process 5812 to run task
[2024-01-17 00:31:53,023] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_redshift', 'verificar_accesos', 'manual__2024-01-17T00:31:47.666927+00:00', '--job-id', '158', '--raw', '--subdir', 'DAGS_FOLDER/ETLSachetto.py', '--cfg-path', '/tmp/tmpzzzi8zwl', '--error-file', '/tmp/tmphx1kfmm6']
[2024-01-17 00:31:53,024] {standard_task_runner.py:80} INFO - Job 158: Subtask verificar_accesos
[2024-01-17 00:31:53,555] {task_command.py:371} INFO - Running <TaskInstance: dag_redshift.verificar_accesos manual__2024-01-17T00:31:47.666927+00:00 [running]> on host 8278623e2516
[2024-01-17 00:31:54,301] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=NachoSachetto
AIRFLOW_CTX_DAG_ID=dag_redshift
AIRFLOW_CTX_TASK_ID=verificar_accesos
AIRFLOW_CTX_EXECUTION_DATE=2024-01-17T00:31:47.666927+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-17T00:31:47.666927+00:00
[2024-01-17 00:31:55,783] {logging_mixin.py:115} INFO - Validaci√≥n Correcto
[2024-01-17 00:31:55,784] {python.py:173} INFO - Done. Returned value was: None
[2024-01-17 00:31:55,807] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_redshift, task_id=verificar_accesos, execution_date=20240117T003147, start_date=20240117T003152, end_date=20240117T003155
[2024-01-17 00:31:55,862] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-17 00:31:56,064] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
